### DISCUSSION TITLE
Remove custom Tensorflow Patch (execution_plan)

### DISCUSSION THREAD

**Discussion by feliwir** [coqui-ai/STT#2210] [General]
The patch making `execution_list` can be removed in favor of the following vanilla API (inside `tflitemodelstate.cpp`):
```cpp
...
  auto orig_plan = interpreter->primary_subgraph().execution_plan();
...
```

In terms of behaviour this is identical to the current code. Since this returns a non-const reference we can also the the execution_plan like that.

**Comment by reuben** [comment]
Can you make a PR?

**Reply by feliwir** [reply]
Can i submit a PR for a CMake based buildsystem first? I refuse to install/use Bazel, i build the native client locally with cmake anyways

**Reply by reuben** [reply]
No, seems like keeping the current code around is a better tradeoff than adding an entire new build system.

**Reply by feliwir** [reply]
Are there any other patches to tensorflow_lite that must be removed?

**Reply by feliwir** [reply]
I can try to apply this fix "blind", but there's little benefit if we can't switch to upstream tensorflow
